templates/content-widget-job_listing.php

```diff
@@ -14,21 +14,29 @@
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly
 }
+
 ?>
 <li <?php job_listing_class(); ?>>
 	<a href="<?php the_job_permalink(); ?>">
-		<div class="position">
-			<h3><?php wpjm_the_job_title(); ?></h3>
+		<?php if( $logo ) { ?>
+		<div class="image">
+			<?php the_company_logo(); ?>
+		</div>
+		<?php } ?>
+		<div class="content">
```

Comment:
Why are we adding <div class="content"> here?

Reply:
The HTML structure had to be changed in order to accommodate the logo added.  I decided to create two div aligned with each other with CSS such that even if a logo is removed the other div will take up space.